www/includes/easyparliament/templates/html/mp/profile.php

```diff
@@ -66,8 +65,8 @@
                 <?php if ( !$current_member[HOUSE_TYPE_COMMONS] ) { ?>
                     <?php if (count($policyPositions->positions) > 0) { ?>
                     <div class="panel">
-                        <a name="votes"></a>
-                        <h2 data-magellan-destination="votes">A selection of <?= $full_name ?>'s votes</h2>
+                        <a name="votes" id="votes"></a>
```

Comment:
Why have both name and ID on these?

Reply:
Oops, yes, not sure why I did this. Jumping to elements by `name` attribute works fine, without an `id`. Force of habit.